Examples/demo-chat-app/demo-chat-app/MessageStore.swift

```diff
@@ -23,33 +23,26 @@ public class MessageStore: ObservableObject {
                 cacheName: momentoClient.cacheName,
                 topicName: momentoClient.topicName
             )
-            switch subResp {
-            case .subscription(let subscription):
-                self.subscription = subscription
-                print("Successful subscription")
-            case .error(let err):
-                fatalError("Unable to establish Topics subscription: \(err)")
+            self.subscription = switch subResp {
+                case .subscription(let sub): sub
+                case .error(let err): fatalError("Unable to establish Topics subscription: \(err)")
```

Comment:
Does this need the same `#if swift(>=5.9)` directive you used below in `example_API_TopicSubscribe` in doc-examples-apis?